modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -42,66 +34,66 @@ Some actions are not yet supported:
 // Reason: RMT and SMT cannot mirror Debian repositories (yet) and so cannot create it in that directory the correct files.
 // We are waiting for SMT to release the feature/fix to mirror Debian repositories. When this has been done, this comment and the limitation above can be removed.
 
+== Prepare {ubuntu} Clients for Registration
```

Comment:
This headline sounds wrong. We do not prepare the clients, we prepare the server to allow registration of these kind of clients.